### ISSUE TITLE
Error at make install - non root-owner

### ISSUE THREAD

**Issue by Goliardo** [abraunegg/onedrive#437]
### Bug Report Details ###
**Describe the bug**
Trying to install (sudo make install), with dmd environment version 2.085.0, a have an error:
`(dmd-2.085.0)goliardo@goliserv:~/onedrive$ sudo make install
echo v2.3.0-1-gebad4ae > version
dmd -w -g -ofonedrive -O  -L-lcurl -L-lsqlite3 -L-ldl -J. src/config.d src/itemdb.d src/log.d src/main.d src/monitor.d src/onedrive.d src/qxor.d src/selective.d src/sqlite.d src/sync.d src/upload.d src/util.d src/progress.d
/var/lib not root-owned 111:0
Makefile:78: recipe for target 'onedrive' failed
make: *** [onedrive] Error 1`

**Application and Operating System Details:**
*   OS: Ubuntu 18.04 Server - Linux goliserv 4.15.0-46-generic #49-Ubuntu SMP Wed Feb 6 09:33:07 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
*   DMD version: DMD64 D Compiler v2.085.0
*   Application configuration: Output of `onedrive --display-config`
*   Curl Version: curl 7.58.0 

Application Log Output:
```bash
goliardo@goliserv:~$ source ~/dlang/dmd-2.085.0/activate
(dmd-2.085.0)goliardo@goliserv:~$ cd onedrive
(dmd-2.085.0)goliardo@goliserv:~/onedrive$ make
echo v2.3.0-1-gebad4ae > version
dmd -w -g -ofonedrive -O  -L-lcurl -L-lsqlite3 -L-ldl -J. src/config.d src/itemdb.d src/log.d src/main.d src/monitor.d src/onedrive.d src/qxor.d src/selective.d src/sqlite.d src/sync.d src/upload.d src/util.d src/progress.d
sed "s|@PREFIX@|/usr/local|g" systemd.units/onedrive.service.in > onedrive.service
sed "s|@PREFIX@|/usr/local|g" systemd.units/onedrive@.service.in > onedrive@.service
sed "s|@DOCDIR@|/usr/local/share/doc/onedrive|g" onedrive.1.in > onedrive.1
(dmd-2.085.0)goliardo@goliserv:~/onedrive$ sudo make install
echo v2.3.0-1-gebad4ae > version
dmd -w -g -ofonedrive -O  -L-lcurl -L-lsqlite3 -L-ldl -J. src/config.d src/itemdb.d src/log.d src/main.d src/monitor.d src/onedrive.d src/qxor.d src/selective.d src/sqlite.d src/sync.d src/upload.d src/util.d src/progress.d
/var/lib not root-owned 111:0
Makefile:78: recipe for target 'onedrive' failed
make: *** [onedrive] Error 1
```

### Bug Report Checklist ###
*   [X] Detailed description
*   [X] Reproduction steps (if applicable)
*   [X] Verbose Log Output

**Comment by abraunegg** [comment]
OK .. 'make install' should not be rebuilding the client. I think this is being triggered by the .PHONY change.

Please can you help validate PR #439:
```
git clone https://github.com/abraunegg/onedrive.git
cd onedrive
git fetch origin pull/439/head:pr439
git checkout pr439
make clean; make;
```

**Comment by abraunegg** [comment]
Makefile issue fixed, v2.3.1 released

**Comment by Goliardo** [comment]
`make install` successful
Thank you!
